src/components/pages/devices/devices.js

```diff
@@ -86,7 +86,8 @@ class DevicesPage extends Component {
             </div>
             <div onClick={this.refreshData} className="refresh-icon icon-sm" />
           </div>
-          <DevicesGrid {...deviceGridProps} />
+          {devices &&
+          <DevicesGrid {...deviceGridProps} pagination={(devices|| []).length > 50 ? true :  false}/>}
```

Comment:
what is "50" in this context and how was the value calculated? could you use a constant instead of hardcoding the value?

Reply:
This value is given by Caitlin. Sure i will use the const instead of hard coding.

Reply:
This value is given by Caitlin. Sure instead of hard coded value i will add const.